app/src/main/AndroidManifest.xml

```diff
@@ -4,7 +4,7 @@
 
     <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
     <application
-        android:allowBackup="true"
+        android:allowBackup="false"
```

Comment:
Any particular reason for this change?

Reply:
I did this for testing. Now I changed it back